react-backend/client/src/components/visitors/button.js

```diff
@@ -1,11 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { PrimaryButtonNoFill } from '../../shared/components/form/base';
 
-const button = props => (
-  <button className="Button" type="submit">
-    {props.label}
-  </button>
-);
+const button = props => <PrimaryButtonNoFill type="submit">{props.label}</PrimaryButtonNoFill>;
```

Comment:
Why not just use `PrimaryButtonNoFill` directly?

Reply:
Yeah, I didn't end up using this component in the end. What's the best way to revert changes from this commit, should I soft reset?